app/client/ui/FormContainer.ts

```diff
@@ -24,7 +24,11 @@ export function buildFormFooter() {
   return [
     cssPoweredByGrist(
       cssPoweredByGristLink(
-        {href: commonUrls.forms, target: '_blank'},
+        {
+          href: commonUrls.forms,
+          target: '_blank',
+          'aria-label': t('Powered by Grist'),
```

Comment:
I'd change it to t(`Powerd by`) + ' Grist'. I don't know how to validate if the translation files are updated correctly.

Reply:
After Florent's feedback I decided to just remove all the added translations from the PR. Will add them in weblate. From my understanding directly updating locale files here messes up with weblate.

But I think it's better to keep "Grist" in the translation so that rtl languages can set up the correct order if needed. Or am I missing something?

Reply:
That's ok.